CMakeLists.txt

```diff
@@ -145,11 +145,18 @@ if(NOT RANGE_V3_NEW_VERSION_HPP STREQUAL RANGE_V3_OLD_VERSION_HPP)
 endif()
 
 include(CMakePackageConfigHelpers)
+
+# write_basic_package_version_file(...) gained ARCH_INDEPENDENT in CMake 3.14.
+# For CMake 3.6, this workaround makes the version file ARCH_INDEPENDENT
+# by unsetting CMAKE_SIZEOF_VOID_P.
+set(OLD_CMAKE_SIZEOF_VOID_P ${CMAKE_SIZEOF_VOID_P})
+unset(CMAKE_SIZEOF_VOID_P)
```

Comment:
Should use `set(CMAKE_SIZEOF_VOID_P)` rather than `unset(...)`: https://cmake.org/cmake/help/latest/command/unset.html#unset-normal-variable-or-cache-entry states:

> [U]nsetting a normal variable can expose a cache variable that was previously hidden. To force a variable reference of the form `${VAR}` to return an empty string, use `set(<variable> "")`, which clears the normal variable but leaves it defined.